gui/webdisplay/src/RWebDisplayHandle.cxx

```diff
@@ -427,7 +429,15 @@ std::string RWebDisplayHandle::ChromeCreator::MakeProfile(std::string &exec, boo
       profile_arg = chrome_profile;
    } else {
       gRandom->SetSeed(0);
-      rmdir = profile_arg = std::string(gSystem->TempDirectory()) + "/root_chrome_profile_"s + std::to_string(gRandom->Integer(0x100000));
+      std::string rnd_profile = "root_chrome_profile_"s + std::to_string(gRandom->Integer(0x100000));
+      profile_arg = gSystem->TempDirectory();
+
+#ifdef _MSC_VER
+      profile_arg += "\\"s + rnd_profile;
+#else
+      profile_arg += "/"s + rnd_profile;
+#endif
```

Comment:
Maybe the `#ifdef _MSC_VER` are not needed. Did you try with forward slashes?

Reply:
Yes, I did try. TempDirectory on Windows returns backslashes and I need to use it, otherwise Firefox complains